app/src/main/java/com/duckduckgo/app/fire/AutomaticDataClearer.kt

```diff
@@ -122,6 +133,11 @@ class AutomaticDataClearer(
         }
     }
 
+    override fun onExit() {
+        // the app does not have any activity in CREATED state we kill the process
+        clearDataAction.killProcess()
```

Comment:
This doesn't seem right. This indiscriminately kills the process regardless of the automatic data clearing settings.

One problem this creates if is you resize the window, then drag it to be full screen again. It triggers the process restart even though (to the user) the app was always in the foreground.